src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/clients/ProfessionalOrganisationRetriever.java

```diff
@@ -73,6 +74,11 @@ public OrganisationEntityResponse retrieve() {
 
         log.info("response returned: {}", response);
 
+        log.info(
+            "PRD endpoint {} : userId[{}] orgId[{}]",
+            refDataApiPath, userDetails.getId(), Objects.requireNonNull(response).getOrganisationIdentifier()
```

Comment:
this may throw NPE when response is null. Returning null at this point is ok. 
You can add userId to the existing log (line 75) and add Lombok `@ToString` annotation to `OrganisationEntityResponse`  so the existing log would be ok

Reply:
Done, thanks.